test/bandit/http1/request_test.exs

```diff
@@ -2159,6 +2192,7 @@ defmodule HTTP1RequestTest do
     end
 
     test "raises an error if client closes while body is being written", context do
+      context = http_server(context, http_options: [log_client_closures: true])
```

Comment:
I'm turning on the option here because we assert that the error happened by capturing the logs, so we need to log the closure in these existing tests.